TestingTravis/TestClass.swift

```diff
@@ -16,7 +16,7 @@ class TestClass: NSObject {
     override init() {
         print(forceUnwrapMe!)
         print(forceUnwrap2!)
-        
+        //testing code owners
         let unused = 0
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/e6554103ad30bf7969c61ea2af2a95d4e5b0f866/TestingTravis/TestClass.swift#L22"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:22</code></td>
    </tr>
  </tbody>
</table>